scripts/prover_profile.sh

```diff
@@ -0,0 +1,24 @@
+
+CIRCUIT=2_2
```

Comment:
Let's document this to say that the variable CIRCUIT takes a value of the form `<JSIN>_<JSOUT>`

Reply:
Refactored the script and added comments to make it slightly more generic.  This has changed as part of that refactor.